addon/components/polaris-empty-state.js

```diff
@@ -55,29 +49,24 @@ export default Component.extend({
    * in which case the block content will be used
    * instead of `text`
    *
-   * @property text
    * @type {String}
    * @default null
    */
-  text: null,
+  text = null;
 
   /**
    * Primary action for empty state
    *
-   * @property action
    * @type {Object}
    * @default null
    */
-  action: null,
+  action = null;
```

Comment:
Does this break in our version of Ember here? I thought we had to update the use of attributes named `action` at some point in our Ember upgrades but maybe I'm remembering wrong.

Reply:
good catch.... it seems to work, but we should probably do this just to avoid confusions
have this noted to recheck once all components are done